pkg/app/env_describe.go

```diff
@@ -135,3 +157,50 @@ func renderDependencies(w io.Writer, dep *types.Dependency) {
 	}
 	table.Render()
 }
+
+type envJsonDescribe struct {
```

Comment:
Maybe we could create a new package pkg/app/formatter to keep the JSON code, WDYT

Reply:
Yes, it would be better to decoupling argument handler and formatter. It's necessary to do this split, thanks.

Reply:
> Maybe we could create a new package pkg/app/formatter to keep the JSON code, WDYT

I have handled the `pkg/app/formatter` and `pkg/app/formatter/json` package
Apart from that, whether it'good to also seal the default table formatter into `pkg/app/formatter/default`?

Reply:
I think so. But the name could be table, WDYT

Reply:
> I think so. But the name could be table, WDYT

Thoungs good, I would work on that these days.